### PR TITLE
Changes made to react router and header

### PR DESCRIPTION
**Pull request by JoelGonzalez02**
-Updated files to pass tests
-Changed react router for home page

Joel Gonzalez

### CODE REVIEW

File: src/components/pages/Home/HomeContainer.js [hunk 1]
```diff
@@ -42,7 +42,8 @@ function HomeContainer() {
           <Spin tip="Loading..." size="large"></Spin>
         </div>
       )}
-      {authState.isAuthenticated && userInfo && <RenderHomePage />}
+      {/* {!authState.isAuthenticated && !userInfo && <RenderHomePage />} */}
```
Comment (tinomen) [inline src/components/pages/Home/HomeContainer.js]: lets not get commented out code into our main branch. Delete the code if it is no longer needed/wanted.

File: src/components/common/Header.js [hunk 2]
```diff
@@ -97,7 +97,14 @@ const Header = () => {
               />
             </a>
             <Divider type="vertical" />
-            <Button onClick={() => authService.logout()}>Logout</Button>
+            {/* <Button onClick={() => authService.logout()}>Logout</Button> */}
```
Comment (tinomen) [inline src/components/common/Header.js]: Remove commented out code.

File: src/components/App.js [hunk 3]
```diff
@@ -33,13 +33,14 @@ const App = () => {
         <Route path="/login" component={LoginPage} />
         <Route path="/implicit/callback" component={LoginCallback} />
 
-        {/* any of the routes you need secured should be registered as SecureRoutes */}
-        <SecureRoute
+        {/* Removed the SecurerRoute component so that the HomePage loads first -JG */}
```
Comment (tinomen) [inline src/components/App.js:36]: What is the driver for this? Couldn't we introduce a new Landing page to be the root path?

**Review by tinomen — CHANGES_REQUESTED** [review]

**Comment by tinomen** [comment]
The description doesn't provide much in the way of motivation for the change to the home page. Also, the changes made to pass test is very unclear. Is that the motivation to change the homepage? If not, lets separate the 2 issues into different PRs.

**Comment by saramatic** [comment]
See Jakes Comments and update what he suggested.

**Review by saramatic — CHANGES_REQUESTED** [review]
Look at Jakes comments. He wants you to be more detailed about what you did on your pull request. And for future PR he wants small individual PR. Once you update what Jake suggest and you push up again I will approve it. :-)

**Comment by saramatic** [comment]
Thanks for updating the routes to show the landing page. And awesome job on passing all the tests from jest

**Review by saramatic — APPROVED** [review]

**Pull request closed without merging**